charts/enterprise-logs/values.yaml

```
@@ -277,7 +277,9 @@ gateway:
   # -- Additional annotations for the `gateway` Pod
   annotations: {}
   # -- Additional labels and annotations for the `gateway` Service
+  # -- Service overriding service type 
```

```suggestion
  # -- Service overriding service type
```
Looks like there's a trailing space here

---

Fixed and pushed the change.